Remove ide (#524)
klaytn/klaytn-docs: docs/getting-started/development-environment.md

**Endpoint Node**

* You [Endpoint Node](../node/endpoint-node/) is needed to connect to the Klaytn network and to issue an API call or send a transaction.
* `ken` is a Klaytn Endpoint Node binary. `ken` exposes two interfaces, a [command-line interface](../node/endpoint-node/ken-cli-commands.md) and the [JSON-RPC APIs](../dapp/json-rpc/). `ken` runs on Linux and MacOS.
* `ken` CLI comes with several utility and node management functions.

**Smart Contract Development**

> * [Remix Online IDE](https://remix.ethereum.org/) - A browser-based compiler and IDE with Klaytn Plugin.

Reviewer: https://ide.klaytn.foundation is now redirected to `remix klaytn plugin site`.
I think the `https://ide.klaytn.foundation` would be better for the users not to find the klaytn plugin of remix.

> * [Remix Online IDE](https://remix.ethereum.org/) - A browser-based compiler and IDE with Klaytn Plugin.

Author: ```suggestion
* [Klaytn Plugin for Remix](https://ide.klaytn.foundation) - Klaytn Plugin for Remix, a browser-based compiler and IDE.
```
